web/src/app/schedules/ScheduleRuleForm.js

```diff
@@ -185,6 +167,15 @@ export default class ScheduleRuleForm extends React.PureComponent {
             <ScheduleTZFilter
               label={tz => `Configure in ${tz}`}
               scheduleID={this.props.scheduleID}
+              onChange={() => {
+                // update value in form container with new TZ
+                onChange({
```

Comment:
Same as above, we shouldn't rely on ScheduleTZFilter change to keep things in sync, the `tz` param is already the source of truth.

Reply:
see above